src/ORM/Table.php

```diff
@@ -1135,6 +1135,9 @@ public function addAssociations(array $params)
      * - finder: The finder method to use when loading records from this association.
      *   Defaults to 'all'. When the strategy is 'join', only the fields, containments,
      *   and where conditions will be used from the finder.
+     * - propertyName: The property name that should be filled with data from the 
+     *   associated table into the source table results. By default this is the underscored 
+     *   & singular name of the association so user in our example.
```

Comment:
Since there is no association config example here in the doc block, this should probably either just explain the inflected format, or provide a short example like:

> for an association alias of `ProductCategories`, this would be `product_category`